internal/tokenizer/tokenize_c.go

```diff
@@ -4,9 +4,10 @@ package tokenizer
 
 import "gopkg.in/src-d/enry.v1/internal/tokenizer/flex"
 
-// Tokenize returns language-agnostic lexical tokens from content. The tokens
-// returned should match what the Linguist library returns. At most the first
-// 100KB of content are tokenized.
+// Tokenize returns lexical tokens from content. The tokens returned match what
```

Comment:
I feel like this comment and the other `Tokenize` comment (in `tokenize.go`) should probably match, up to the specific differences (viz., the known bug in the above).